### PR TITLE
Add fonts; configure webpack; build new login page

### PR DESCRIPTION
**Pull request by kevinkassimo**
## Description

__DO NOT MERGE__: this is just to make sure that people know the demo login page has been set up
## Make sure
- [x] Tested and Working

## Test Plan

screen shot 2018-05-01 at 12 32 32 am

### CODE REVIEW

File: src/components/pages/login/index.js [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
     render() {
         return(
             <div className="container-login">
                 <div className="logo">
-                    <img src={logo} id="logo" alt="logo" />
+
+					{/*<img className="logo__logo" src={newBquestLogo} alt="newBquestLogo"/>*/}
+					{/*<img className="logo__text" src={newBquest} alt="newBquest"/>*/}
+					<div className="logo__logo" dangerouslySetInnerHTML={{__html: newBquestLogo}} />
+					<div className="logo__text" dangerouslySetInnerHTML={{__html: newBquest}} />
```
Comment (ramsgoli) [inline src/components/pages/login/index.js]: please fix indenting here. I thought that the .editorconfig file in the root of our repo would configure your/all text editors to obey this indenting but i guess not. I'll try to figure out why

File: src/components/pages/login/index.js [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
                 </div>
                 <div className="container-loginform">
-                    Create an account <Link className="register-color" to="/register">here</Link> if you do not have one
+					<h1 className="loginform__title">Login</h1>
```
Comment (ramsgoli) [inline src/components/pages/login/index.js]: just for consistency, please keep classNames with regular en-dash, rather than underscores

Comment (kevinkassimo) [reply src/components/pages/login/index.js]: Okay. (I originally want to probably comply with BEM model though)

File: src/components/pages/login/index.js (hunk 1 above)
Comment (kevinkassimo) [reply src/components/pages/login/index.js]: Seems Webstorm is not recognizing its file type... I'll see if I can fix it...

Comment (kevinkassimo) [reply src/components/pages/login/index.js]: Actually, I think the indenting here should be correct: I checked the file itself and seems indenting correctly, it is just at Github's comparison that makes it look longer

**Pull request merged**